src/renderer/pages/app.vue

```diff
@@ -186,6 +190,9 @@ export default {
     }, false)
 
     this.$nextTick(() => {
+      this.$watch('zoom', function (zoom) {
```

Comment:
This should be moved to the `watch` property above.

Reply:
fixed